svthalia/concrexit
# Update header photos

Closes #2318 #3461 .

### Summary
- Changed headers on standard pages by adding a few banners and deleting old ones
- Photos pages now have random headers as well, but from a different subset of headers

Main philosophy is that the photos section (being behind a login) can have fun photos of events, whereas most pages of the website should have more serious photos because everyone can see those. 

### How to test
1. Open website
2. Refresh random pages to see if all photos work 
3. Go to the photos pages, refresh multiple times to see if all photos work
